### PR TITLE
1101.feature Add verbose flag to check command

### PR DESCRIPTION
**Pull request by jwjacobson**
Trac 1101 suggests adding the verbose flag to every command for better error handling, even if there is no difference in output. @meejah Please suggest improvements to this PR or advice on how I should do it differently

### CODE REVIEW

**Review by meejah — APPROVED** [review]
LGTM

(Sorry, been waiting on "fixing CI" somewhat before merging, but this looks good as is)

**Comment by meejah** [comment]
Sorry this has been sitting a while, if you merge "master" in more CI should turn green ... :crossed_fingers:

**Comment by blaisep** [comment]
@meejah , I helped @jwjacobson merge with `master` so if you notice anything amiss, let me know and I won't steer him wrong the next time.

**Pull request merged**
